src/features/bivariate_manager/components/BivariateMatrixContainer/BivariateMatrixContainer.tsx

```diff
@@ -57,28 +61,27 @@ const BivariateMatrixContainer = ({ className }: BivariateMatrixContainerProps)
         className={clsx(s.bivariateContainer, className)}
         ref={containerRef}
       >
-        <div>
-          {statesToComponents({
-            loading: (
-              <div className={s.loadingContainer}>
-                <LoadingSpinner />
-              </div>
-            ),
-            error: () => (
-              <div className={s.errorContainer}>
-                <ErrorMessage message="Unfortunately, we cannot display the matrix. Try refreshing the page or come back later." />
-              </div>
-            ),
-            ready: () => (
-              <>
-                <div className={s.matrixContainer}>
-                  <ConnectedBivariateMatrix ref={onRefChange} />
-                </div>
-                <BivariateGreetingsContainer className={s.greetings} />
-              </>
-            ),
-          })}
-        </div>
+        {statesToComponents({
+          loading: (
+            <div className={s.loadingContainer}>
+              <LoadingSpinner />
+            </div>
+          ),
+          error: () => (
+            <div className={s.errorContainer}>
+              <ErrorMessage message="Unfortunately, we cannot display the matrix. Try refreshing the page or come back later." />
```

Comment:
is this translatable?

Reply:
Nope, ErrorMessage just do some stylings, will add! thanks for the catch!